src/librustc/lint/mod.rs

```diff
@@ -26,7 +26,7 @@ use rustc_data_structures::sync::{self, Lrc};
 use crate::hir::def_id::{CrateNum, LOCAL_CRATE};
 use crate::hir::intravisit;
 use crate::hir;
-use crate::lint::builtin::{BuiltinLintDiagnostics, DUPLICATE_MATCHER_BINDING_NAME};
+use crate::lint::builtin::{self, BuiltinLintDiagnostics, DUPLICATE_MATCHER_BINDING_NAME};
```

Comment:
ugh, I'm sorry, I broke this code thinking I could make a seemingly trivial fix.

I'll do a proper fix and kill off this bad commit in the process.

Reply:
Decided on:

```rust
use crate::lint::builtin as lint_builtin;
```

am I proud of this? No. No I am not.